main.go

```diff
@@ -30,7 +31,6 @@ import (
 	operatorv1 "github.com/opencurve/curve-operator/api/v1"
 	"github.com/opencurve/curve-operator/pkg/clusterd"
 	"github.com/opencurve/curve-operator/pkg/controllers"
-	"github.com/spf13/pflag"
```

Comment:
Why change position the package? grouping by 'github.com' should be more formal

Reply:
It's clearer to group local packages alone. What do you think?

Reply:
yeah, you are right. I checked my tool and it's not work.

Reply:
![Uploading Screenshot 2023-04-12 at 16.09.13.png…]()

If you use GoLand, you can check `Group` and select `Current project packages`.

Reply:
thanks, but vscode is for me. :)

Reply:
> thanks, but vscode is for me. :)

I see. Let IDEs modify that.